windows/frescobaldi-wininst.py

```diff
@@ -41,7 +41,7 @@
 
 
 # Main application script and Python executable:
-script = os.path.join(sys.prefix, 'Scripts', 'frescobaldi')
+script = os.path.join(sys.prefix, 'Scripts', 'frescobaldi', '__main__.py')
```

Comment:
I believe this should be `os.path.join(sys.prefix, 'Scripts', 'frescobaldi_app', '__main__.py')`, but I'm not sure (where does `Scripts` come from?) and cannot test on Windows now.